avaframe/ana4Stats/probAna.py

```diff
@@ -19,6 +18,102 @@
 log = logging.getLogger(__name__)
 
 
+def createComModConfig(cfgProb, avaDir, modName):
+    """ create configuration file for performing sims with modName com module
+
+        Parameters
+        -----------
+        cfgProb: configParser object
+            configuration settings
+        avaDir: pathlib path
+            path to avalanche directory
+        modName: module
+            computational module
+
+        Returns
+        -------
+        cfgFiles: dict
+            dictionary of paths to newly generated configuration files for com module for all parameters 
+
+    """
+
+    # setup where configuration file is saved
+    outDir = avaDir / 'Outputs'
+    fU.makeADir(outDir)
+
+    # loop over all parameters for performing parameter variation
+    varParList = cfgProb['PROBRUN']['varParList'].split('|')
+    cfgFiles = {}
+    for varName in varParList:
+        # define configuration files
+        cfgFile = outDir / ('probRuncom1DFACfg%s.ini' % varName)
+
+        # use default com module settings or local settings
+        if cfgProb['PROBRUN'].getboolean('defaultSetup'):
+            com1DFACfg = cfgUtils.getDefaultModuleConfig(modName)
+            com1DFACfg, refIn = updateCfgRange(com1DFACfg, cfgProb, varName)
+            with open(cfgFile, 'w') as configfile:
+                com1DFACfg.write(configfile)
+        else:
+            com1DFACfg = cfgUtils.getModuleConfig(modName)
+            com1DFACfg, refIn = updateCfgRange(com1DFACfg, cfgProb, varName)
+            with open(cfgFile, 'w') as configfile:
+                com1DFACfg.write(configfile)
```

Comment:
The function name gives the impression of being for any com Module, but here com1DFA is used in naming. I suggest to rename this variable.

Reply:
renamed and also the output cfg file is using the modName